rca/static_src/javascript/programmes/programmes.types.js

```diff
@@ -1,24 +1,35 @@
 import PropTypes from 'prop-types';
 
+export const PROGRAMME_PAGE_TYPE = 'programmes.ProgrammePage';
+export const SHORT_COURSE_PAGE_TYPE = 'shortcourses.ShortCoursePage';
+
 /**
- * A Programme page’s serialised attributes, coming from the Wagtail API.
- * Please make sure this stays up-to-date with APIField and serializers.
+ * A ProgrammePage or ShortCoursePage’s serialised attributes, coming from the Wagtail API.
+ * Please make sure this stays up-to-date with api_field definitions and serializers.
  */
 export const programmePage = {
     id: PropTypes.number.isRequired,
     meta: PropTypes.shape({
-        type: PropTypes.string.isRequired,
+        type: PropTypes.oneOf([PROGRAMME_PAGE_TYPE, SHORT_COURSE_PAGE_TYPE])
```

Comment:
Feels a bit overkill but at least it makes it clear this is now for two different page types.